### PR TITLE
zh-translation: /blog/2019/egress-traffic-control-in-istio-part-2/index.md

### PR DESCRIPTION
**Pull request by helight**
ref: https://github.com/servicemesher/istio-official-translation/issues/1128

### CODE REVIEW

File: content/zh/blog/2019/egress-traffic-control-in-istio-part-2/index.md [hunk 1]
```diff
@@ -1,129 +1,80 @@
...
+- 允许应用程序 **B** 访问 `mongo1.composedb.com`。
+- 监控所有的出口流量。
+
+假设攻击者有以下目标：
+
+- 从你的集群中访问 `*.ibm.com`。
+- 从你的集群中访问 `*.ibm.com`，并且不被监控到。攻击者不想他的流量被监控到，如果被监控到你将会发觉这个禁止的访问。
+- 从你的集群中访问 `mongo1.composedb.com`。
+
+现在假设攻击者设法在攻破应用程序 **A** 的其中一个 pod，并且试图使用这个攻破的 pod 来执行被禁止的访问。攻击者可能试试运气直接访问外部服务。你会对这个直接的尝试做出如下反应：
```
Comment (gorda) [inline content/zh/blog/2019/egress-traffic-control-in-istio-part-2/index.md]: > 这个攻破的 pod 

建议：这个被攻破的 pod

File: content/zh/blog/2019/egress-traffic-control-in-istio-part-2/index.md [hunk 2]
```diff
@@ -1,129 +1,80 @@
 ---
-title: Secure Control of Egress Traffic in Istio, part 2
-subtitle: Use Istio Egress Traffic Control to prevent attacks involving egress traffic
-description: Use Istio Egress Traffic Control to prevent attacks involving egress traffic.
+title: Istio 中的安全管控出口流量，第二部分
+subtitle: 使用 istio 的出口流量管控来阻止相关出口流量攻击
+description: 使用 istio 的出口流量管控来阻止相关出口流量攻击。
```
[the remark this answers is not in the document]
Comment (helight) [reply content/zh/blog/2019/egress-traffic-control-in-istio-part-2/index.md]: accept

File: content/zh/blog/2019/egress-traffic-control-in-istio-part-2/index.md [hunk 3]
```diff
@@ -1,129 +1,80 @@
...
+欢迎来看 Istio 对出口流量进行安全管控系列文章的第 2 部分。
+在[这个系列文章的第一部分](/zh/blog/2019/egress-traffic-control-in-istio-part-1/),我提出了出口流量相关攻击和针对出口流量进行安全管控我们收集的要求点。
+在这一期中，我会讲述对出口流量进行安全管控的 Istio 方式，并且展示 Istio 如何帮你阻止攻击。
+
+## Istio 中的出口流量安全管控 {#secure-control-of-egress-traffic-in-Istio}
+
+为了在 Istio 中实施出口流量的安全管控，你必须[通过出口网关将 TLS 流量发送到外部服务](/zh/docs/tasks/traffic-management/egress/egress-gateway/#egress-gateway-for-https-traffic)。
+或者可以[通过出口网关发送 HTTP 流量](/zh/docs/tasks/traffic-management/egress/egress-gateway/#egress-gateway-for-http-traffic)，并且[让出口网关来发起执行 TLS](/zh/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/#perform-TLS-origination-with-an-egress-gateway)。
+
+两种选择各有利弊，你应该根据你的具体场景进行选择。选择的关键取决于你的应用陈旭是否能发送不加密的 HTTP 请求和你团队的安全策略是否允许发送不加密的 HTTP 请求。
```
[the remark this answers is not in the document]
Comment (helight) [reply content/zh/blog/2019/egress-traffic-control-in-istio-part-2/index.md]: accept

File: content/zh/blog/2019/egress-traffic-control-in-istio-part-2/index.md [hunk 4]
```diff
@@ -1,129 +1,80 @@
...
+例如，如果你的应用程序使用了某些客户端库，用这些库来对流量进行加密，但是它无法取消加密，你就不能使用发送不加密 HTTP 流量的选项。万一你团队的安全策略无法让你**在 pod 内**发送不加密的 HTTP 请求也是一样的（pod 外的流量由 Istio 来加密）。
+
+如果应用程序发送 HTTP 请求，并且由出口网关发起执行 TLS，你就可以监控 HTTP 信息，像 HTTP 方法、HTTP 头和 URL 路径。也可以根据上面说的 HTTP 信息来[定义策略](/zh/blog/2018/egress-monitoring-access-control)。如果是由应哟过程序发起执行 TLS，你就可以对源 pod 的 TLS 流量的 [SNI 和服务账号进行监控](/zh/docs/tasks/traffic-management/egress/egress_sni_monitoring_and_policies/)，并且基于 SNI 和服务账号定义策略。
```
[the remark this answers is not in the document]
Comment (helight) [reply content/zh/blog/2019/egress-traffic-control-in-istio-part-2/index.md]: accept

File: content/zh/blog/2019/egress-traffic-control-in-istio-part-2/index.md [hunk 5]
```diff
@@ -1,129 +1,80 @@
...
+
+你必须确保你集群到外部的流量不能绕过出口网关。Istio 不能给你确保这一点，所以你必需使用一些[附加的安全机制](/zh/docs/tasks/traffic-management/egress/egress-gateway/#additional-security-considerations)，比如 [Kubernetes 网络策略](https://kubernetes.io/docs/concepts/services-networking/network-policies/)或者 L3 防火墙。 看一个 [Kubernetes 网络策略配置](/zh/docs/tasks/traffic-management/egress/egress-gateway/#apply-Kubernetes-network-policies)的例子。
+根据[纵深防御](https://en.wikipedia.org/wiki/Defense_in_depth_(computing)) 的概念，为同一个目标使用的安全机制越多越安全。
+
+你必需也要确保 Istio 控制面和出口网关不能被破坏。你的集群里面可能有成百上千的应用程序 pod，而只有十几个 Istio 控制面 pod 和网关。
```
[the remark this answers is not in the document]
Comment (helight) [reply content/zh/blog/2019/egress-traffic-control-in-istio-part-2/index.md]: accept，thank you

File: content/zh/blog/2019/egress-traffic-control-in-istio-part-2/index.md (hunk 1 above)
Comment (helight) [reply content/zh/blog/2019/egress-traffic-control-in-istio-part-2/index.md]: accept

**Comment by helight** [comment]
> @helight 翻译得很棒，我只能挑点字眼错误，哈哈
感谢感谢

**Comment by tanjunchen** [comment]
@rootsongjc 
add translation/chinese lavel for calculating data.

**Pull request merged**